product_docs/docs/eprs/7/07_common_operations/09_offline_snapshot.mdx

```diff
@@ -136,10 +143,18 @@ To create a primary node from an offline snapshot:
 
 1.  If you modified the publication server configuration file in step 3, reload configuration of the publication server. See ["Reloading the Publication or Subscription Server Configuration File (reloadconf)" ](../05_smr_operation/02_creating_publication/01_registering_publication_server/#registering_publication_server) for directions to reload the publication server's configuration.
 
-1.  In the database to use as the new primary node, create the schema and the table definitions, and load the tables from your offline data source.
+### Adding primary nodes 
+
+Execute these steps to add a primary node from an offline snapshot. Repeat them for each additional primary node.
 
 1.  Add the primary node as described in [Creating more primary nodes](../06_mmr_operation/03_creating_primary_nodes/#creating_primary_nodes) with the options **Replicate Publication Schema** and **Perform Initial Snapshot** cleared.
 
+1.  In the database to use as the new primary node, create the schema and the table definitions, and load the tables from your offline data source.
+
+    !!!note
+    Ensure you synchronize the offline data source from the source Publication database after you have added the target node in the MMR cluster. Otherwise, certain changes from the source database won't be replicated.
```

Comment:
As suggested for SMR, better to replace "synchronize" with "populate".

Reply:
I used the word "load" as it was being used throughout the document. I also rephrased a bit. Do you mind taking a look and letting me know what you think?